run_dir/static/js/running_notes_component.js

```diff
@@ -0,0 +1,557 @@
+            }
+
+            let diffInHours = Math.floor(diffInMinutes / 60);
+            if (diffInHours < 24) {
+                return `${diffInHours} hours ago`;
+            }
+
+            let diffInDays = Math.floor(diffInHours / 24);
+            return `${diffInDays} days ago`;
+        },
+        formatted_note() {
+            if (this.note == undefined) {
+                return ''
+            }
+            return marked.parse(this.note)
+        },
+        running_note() {
+            if (this.running_note_obj == undefined) {
+                return undefined
+            }
+            // Check if running note is an object
+
+            if (typeof this.running_note_obj == 'object') {
+                return this.running_note_obj
+            }
+            let running_note_json = JSON.parse(this.running_note_obj)
+            return Object.values(running_note_json)[0];
+        },
+        email() {
+            return this.getRunningNoteProperty('email')
+        },
+        note() {
+            return this.getRunningNoteProperty('note')
+        },
+        user() {
+            return this.getRunningNoteProperty('user')
+        }
+    },    
+    methods: {
+        generate_category_label(categories){
+            let cat_classes = {
+               'Workset': ['primary', 'calendar-plus'],
+               'Flowcell': ['success', 'grip-vertical'],
+               'Decision': ['info', 'thumbs-up'],
+               'Lab': ['succe', 'flask'],
+               'Bioinformatics': ['warning', 'laptop-code'],
+               'User Communication': ['usr', 'people-arrows'],
+               'Administration': ['danger', 'folder-open'],
+               'Important': ['imp', 'exclamation-circle'],
+               'Deviation': ['devi', 'frown'],
+               'Invoicing': ['inv', 'file-invoice-dollar'],
+               'Sticky': ['sticky', 'note-sticky']
+           }
+           let cat_label = '';
+           Object.values(categories).forEach(function(val){
+             var cat = val.trim()
+             if (Object.keys(cat_classes).indexOf(cat) != -1){
+                 cat_label += '<span class="badge bg-'+cat_classes[cat][0]+'">'+cat+'&nbsp;'+'<span class="fa fa-'+ cat_classes[cat][1] +'">'+"</span></span> ";
+             }
+           });
+           return cat_label;
+        },
+        getRunningNoteProperty(key){
+            if (this.running_note !== undefined) {
+                if (key in this.running_note) {
+                    return this.running_note[key]
+                }
+            }
+            return undefined
+        }
+    },
+    template:
+    /*html*/`
+    <div class="pb-3">
+        <div class="card">
+            <div class="card-header bi-project-note-header">
+                <a class="text-decoration-none" :href="'mailto:' + this.email">{{this.user}}</a>
+                <template v-if="!compact">
+                - <span class="todays_date">{{ formattedTimeStamp }}</span>
```

Comment:
I've looked at it a little bit but its probably some work to add back the card ids so that we can link to individual cards

Reply:
Not entirely sure what you're trying to accomplish. Linking from a running note to a specific project card?

Reply:
I think we resolved this in the PR you made as well right?

Reply:
Yep, all done!